src/renderer/shared/values/tokens.js

```diff
@@ -0,0 +1,137 @@
+export const ACAT = '7f86d61ff377f1b12e589a5907152b57e2ad9a7a';
+export const APH = 'a0777c3ce2b169d4a23bcba4565e3225a0122d95';
+export const APT = '7cd338644833db2fd8824c410e364890d179e6f8';
+export const ASA = 'a58b56b30425d3d1f8902034996fcac4168ef71d';
+export const AVA = 'de2ed49b691e76754c20fe619d891b78ef58e537';
+export const BCS = '546c5872a992b2754ef327154f4c119baabff65f';
+export const CGE = '34579e4614ac1a7bd295372d3de8621770c76cdc';
+export const CPX = '45d493a6f73fa5f404244a5fb8472fc014ca5885';
+export const DBC = 'b951ecbbc5fe37a9c280a76cb0ce0014827294cf';
+export const EDS = '81c089ab996fc89c468a26c0a88d23ae2f34b5c0';
+export const EFX = 'acbc532904b6b51b5ea6d19b803d78af70e7e6f9';
+export const EXT = 'e8f98440ad0d7a6e76d84fb1c3d3f8a16e162e97';
+export const GALA = '9577c3f972d769220d69d1c4ddbd617c44d067aa';
+export const GDM = 'd1e37547d88bc9607ff9d73116ebd9381c156f79';
+export const IAM = '891daf0e1750a1031ebe23030828ad7781d874d6';
+export const LRN = '06fa8be9b6609d963e8fc63977b9f8dc5f10895f';
+export const MCT = 'a87cc2a513f5d8b4a42432343687c2127c60bc3f';
+export const NKN = 'c36aee199dbba6c3f439983657558cfb67629599';
+export const NRVE = 'a721d5893480260bd28ca1f395f2c465d0b5b1c2';
+export const OBT = '0e86a40588f715fcaf7acd1812d50af478e6e917';
+export const ONT = 'ceab719b8baa2310f232ee0d277c061704541cfb';
+export const PKC = 'af7c7328eee5a275a3bcaee2bf0cf662b5e739be';
+export const QLC = '0d821bd7b6d53f5c2b40e217c6defc8bbe896cf5';
+export const RHT = '2328008e6f6c7bd157a342e789389eb034d9cbc4';
+export const TKY = '132947096727c84c7f9e076c90f08fec3bc17f18';
+export const RPX = 'ecc6b20d3ccac1ee9ef109af5a7cdb85706b1df9';
+export const SOUL = 'ed07cffad18f1308db51920d99a2af60ac66a7b3';
+export const SWTH = 'ab38352559b8b203bde5fddfa0b07d8b2525e132';
+export const THOR = '67a5086bac196b67d5fd20745b0dc9db4d2930ed';
+export const TNC = '08e8c4400f1af2c20c28e0018f29535eb85d15b6';
+export const XQTA = '6eca2c4bd2b3ed97b2aa41b26128a40ce2bd8d1a';
+export const ZPT = 'ac116d4b8d4ca55e6b6d4ecce2192039b51cccc5';
```

Comment:
Hardcoding these takes us down a path that makes use dependent upon releases every time there's a new NEP5 token, and I don't want us to have to worry about that as Neon Wallet has had to.  There's a [neo-tokens repo](https://github.com/CityOfZion/neo-tokens) that we can use to fetch the list of tokens whenever the app runs, and use that instead.  I would like for us to do that so that there is never any manual updating required in here for tokens.

Along with such a change, we won't ever need this constants file.

Reply:
what would be best way to use that neo-tokens package? as far as I can see it's not on yarn/npm

Reply:
We can just fetch the JSON file directly.  https://raw.githubusercontent.com/CityOfZion/neo-tokens/master/tokenList.json

Reply:
fixed